src/syscall/x86_64.rs

```diff
@@ -13,20 +13,18 @@
 // option is specified.
 use core::arch::asm;
 
-use crate::arch::x86_64::Sysno;
-
 /// Issues a raw system call with 0 arguments.
 ///
 /// # Safety
 ///
 /// Running a system call is inherently unsafe. It is the caller's
 /// responsibility to ensure safety.
 #[inline]
-pub unsafe fn syscall0(n: Sysno) -> usize {
+pub unsafe fn raw_syscall0(n: usize) -> usize {
```

Comment:
I don't think these names need to change to `raw_*` actually. All architecture-specific syscalls are private (but reexported via `syscalls::raw::*`).

Keeping these the same should result in the least amount of breaking change. Likewise, the `syscalls::raw::*` names should remain the same and where the only difference is that `Sysno` is replaced by `usize`.